static/docs/commands-reference/destroy.md

```diff
@@ -39,15 +39,57 @@ $ echo foo > foo
 $ dvc add foo
 $ ls -a
 
-.dvc .git code.py foo foo.dvc
+.dvc .git foo foo.dvc
 
 $ dvc destroy
 
-This will destroy all information about your pipelines as well as cache in .dvc/cache.
-Are you sure you want to continue?
+This will destroy all information about your pipelines, all data files, as well
```

Comment:
this is a pre-formatted block. No need to wrap the line here. It should be exactly like DVC otputs